Makefile

```diff
@@ -182,7 +188,10 @@ $(SCRIPTS_INSTALL_DIR) :
 
 install_common_scripts : $(SCRIPT_COMMON_INSTALL_DIR) $(COMMON_SCRIPTS_SRC)
 	$(CP) $(COMMON_SCRIPTS_SRC)/*.sh $(SCRIPT_COMMON_INSTALL_DIR)
-	
+
+install_core_scripts : $(SCRIPT_CORE_INSTALL_DIR) $(CORE_SCRIPTS_SRC)
```

Comment:
Why not follow the example set by install_postgres_script  where we specify the single script we want to copy over? I think rsync is a valid method, but for makefile readability and consistency, I would prefer to use set patterns (install_postgres_script for example). If we switch to use rsync, I would prefer all similar targets to use rsync (for the sake of consistency), and it should be done in a separate issue/PR.